tests/foreman/sys/test_katello_certs_check.py

```diff
@@ -201,6 +202,7 @@ def test_positive_update_katello_certs(self, cert_setup, cert_data):
                 result = connection.run('foreman-maintain health check --label services-up -y')
                 assert result.return_code == 0, 'Not all services are running'
 
+    @pytest_skip_if_bug_open('bugzilla', 1899108)
```

Comment:
I think you should use `pytest.mark.skip_if_open('BZ:1899108')`, which does not need to be imported and is more consistent with the test of the test modules.  Is there a specific robozilla behavior that's required here?